python/invoke_release/tasks.py

```diff
@@ -1359,7 +1363,16 @@ def release(_, verbose=False, no_stash=False):
 
         if USE_PULL_REQUEST:
             _checkout_branch(verbose, current_branch_name)
-
+            try:
+                os.environ["GITHUB_TOKEN"]
+            except KeyError:
+                _standard_output("$GITHUB_TOKEN not set. Can't open PR")
```

Comment:
do we have any documentation url for this or official github documentation

Reply:
https://developer.github.com/v3/pulls/#create-a-pull-request